src/api-v1.js

```diff
@@ -103,10 +106,17 @@ api.declare({
             url: url,
           });
         }
+        // TODO: Should only status === 'error' be considered a miss?  A file
+        // that's never been requested before could never be in the cache, so
+        // it not being there really is not an error case
+        this.monitor.count(`${service}.${region}.cache-miss`, 1);
```

Comment:
no worries.  For now, i'd like to use the string interpolation version as I know it works and it's only a nit, that ok?